uzupelnianka.txt

```diff
@@ -1,7 +1,7 @@
-Wyjątki dzielimy na … które trzeba obsłużyć i … które można, ale nie trzeba obsłużyć.
+Wyjątki dzielimy na kontrolowane które trzeba obsłużyć i niekontrolowane które można, ale nie trzeba obsłużyć.
 
-Wyjątki … dziedziczą po klasie RuntimeException
+Wyjątki exception dziedziczą po klasie RuntimeException
 
-Wyjątki rzucamy korzystając z instrukcji …, jeśli rzucamy wyjątek kontrolowany musimy dodatkowo w sygnaturze metody/konstruktora zadeklarować wyjątek używając słowa …
+Wyjątki rzucamy korzystając z instrukcji throw new Exception(), jeśli rzucamy wyjątek kontrolowany musimy dodatkowo w sygnaturze metody/konstruktora zadeklarować wyjątek używając słowa throws Exception
```

Comment:
- throw new Exception() - instrukcja to samo `throw`
- throws Exception -  samo `throws`